src/dart-2.md

```diff
@@ -63,7 +63,7 @@ from either Dart 1.x or an earlier version of Dart 2.
    * [Flutter SDK upgrade instructions][Flutter SDK upgrade]
    * [Dart SDK instructions][Dart SDK install] (server-side or web)
 2. **Upgrade the packages your app depends on.**
-   * Flutter: [`flutter packages upgrade`][flutter package upgrade]
+   * Flutter: [`flutter pub upgrade`][flutter package upgrade]
```

Comment:
Can you upgrade the link, too? As in:
   * Flutter: [`flutter pub upgrade`][flutter package upgrade]
  =>
   * Flutter: [`flutter pub upgrade`][flutter pub upgrade]

Reply:
Done!